tide/wall/datasources/PDFTiler.cpp

```diff
@@ -55,11 +55,21 @@ namespace
 const uint tileSize = 2048;
 }
 
-PDFTiler::PDFTiler(const QString& uri)
-    : LodTiler{PDF{uri}.getSize() * VectorialContent::getMaxScale(), tileSize}
-    , _uri{uri}
-    , _tilesPerPage{_lodTool.getTilesCount()}
+PDFTiler::PDFTiler(const QString& uri, const QSize& maxImageSize)
+    : _uri{uri}
 {
+    try
+    {
+        std::make_unique<PDF>(uri);
```

Comment:
that's really just creating a temporary object to check if fails?

Reply:
```_pdf = ``` missing?

Reply:
yes, it's checking if the pdf can be opened, no _pdf missing